Spring-JMS/target/classes/application.properties

```diff
@@ -0,0 +1,134 @@
+# (c) Copyright IBM Corporation 2021
```

Comment:
The target directory is built, consequently nothing from target should be in the repo.